src/components/Offramp/Confirm.view.tsx

```
@@ -665,6 +725,76 @@
         }
     }
 
+    const calculateBankingFee = useCallback(
+        (accountType: string) => {
+            if (!!appliedPromoCode) return 0
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)